docs/FAQs.md

```diff
@@ -0,0 +1,37 @@
+# FAQs
+## Allocation
+### How should I define scheduling constraints?
+Karpenter takes a layered approach to scheduling constraints. Each Cloud Provider has its own set of global defaults, which are overriden by defaults specified in the Provisoner, which are overriden by Pod scheduling constraints. This model requires minimal configuration for most use cases, and supports diverse workloads using a single Provisioner.
+### Does Karpenter replace the Kube Scheduler?
+No. Provisioners work in tandem with the Kube Scheduler. When capacity is unconstrained, the Kube Scheduler will schedule pods as usual. It may schedule pods to nodes managed by Provisioner or other types of capacity in the cluster. Provisioners only attempt to schedule pods of when `type=PodScheduled,reason=Unschedulable`. In this case, they will make a provisioning decision, launch new capacity, and bind pods to the provisioned nodes. Provisioners do not wait for the Kube Scheduler to make a scheduling decision in this case, as the decision is already made by nature of making a provisioning decision.
```

Comment:
typo `of when`?

Reply:
I think it might make sense to add  a question around "does Karpenter race with the Kube Scheduler" (and/or address in this question?)

Reply:
Good point